firmware/rhizome.ino

```diff
@@ -98,20 +111,32 @@ void setup() {
  * The meat of the program. Runs repeatedly until the Rhizome is powered off.
  */
 void loop() {
+    //probe ID finder command (switch this for the other below commands to enable)
+//    ((Ohmbrewer::Onewire*)(((Ohmbrewer::RIMS*)sprouts.front())->getTube()->getSensor()->getProbe()))
+//            ->displayProbeIds(&screen);
+//    screen.println( ((Ohmbrewer::HeatingElement*)sprouts.front())->getControlPin() );
+//    screen.println( ((Ohmbrewer::HeatingElement*)sprouts.front())->getPowerPin() );
+// int a = 0;
+//    if (a==0){
+//        sprouts.push_back(new Ohmbrewer::Thermostat(1, thermPins));
+//        a++;
+//    }
+    screen.println(thermPins->front());
```

Comment:
this was just a test to make sure the list was passed throguh the constructors and popped correctly